src/browser.ts

```diff
@@ -50,6 +45,7 @@ const MY_BINDER_DISABLED = 'jp-MyBinderButton-disabled';
  */
 export
 class GitHubFileBrowser extends Widget {
+  gitHubBaseUrl: string;
```

Comment:
By doing this you have made it a public property. If there is no reason for it to be exposed in the API, I would prefer this to be private. You can do this with
```typescript
  private _gitHubBaseUrl: string
```
By convention, private members are declared at the bottom of the class and prefixed with an underscore.

Reply:
Since it's currently set externally from the `activateFileBrowser` function it would seem this would have to be public? Unless perhaps we went with the dynamic setting?